.github/workflows/languages-summary.yml

```diff
@@ -0,0 +1,27 @@
+name: Languages summary
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'languages/*/config.json'
+      - 'languages/*/exercises/**'
```

Comment:
I think this might be missing a trailing *? Would have to check the other workflows to be sure though

Reply:
I've borrowed this from the `julia-exercise-ci.yml` workflow and it seems to work for the stories one too.